components/raftstore/src/store/config.rs

```diff
@@ -29,8 +29,9 @@ with_prefix!(prefix_store "store-");
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct Config {
-    // true for high reliability, prevent data loss when power failure.
-    pub sync_log: bool,
```

Comment:
This breaks backward compatibility. Cluster that disabled `sync-log` will enable it again and cause high latency after upgrading.

Reply:
Yes, it's on purpose. Do you have a better idea? Keep this config?

Reply:
I suggest to keep it on release-4.0 and remove it on master. After we think the feature is stable, we can enable fsync control when sync-log is set to false.

/cc @innerr @zhangjinpeng1987